tasks/collect-pyxis-params/tests/pre-apply-task-hook.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+# Create a dummy pyxis secret (and delete it first if it exists)
+kubectl delete secret test-collect-pyxis-params-cert --ignore-not-found
+kubectl create secret generic test-collect-pyxis-params-cert --from-literal=cert=mycert --from-literal=key=mykey
```

Comment:
Why is this needed? The task doesn't actually do anything with the secret - that happens in the pyxis tasks.

Reply:
Good catch. This is leftover from when I was leaning in a different direction. Thanks